scalatest/src/main/scala/org/scalatest/tools/Framework.scala

```diff
@@ -430,16 +431,17 @@ class Framework extends SbtFramework {
           annotationClass.getName
         else
           value
-      }
+      }).asInstanceOf[Array[String | Null]]
     
-    def execute(eventHandler: EventHandler, loggers: Array[Logger]) = {
+    def execute(eventHandler: EventHandler | Null, _loggers: Array[Logger | Null] | Null) = {
+      val loggers = _loggers.nn.map(_.nn)
```

Comment:
**Overridden java arguments are never null**
Argument should never be null, immediately remove nullness of parameter.